doc/index.rst

```diff
@@ -90,6 +90,7 @@ Catalyst
    demos/adaptive_circuits_demo
    demos/qml/tutorial_qubit_rotation
    QML Optimization with Optax <demos/qml_optimization>
+   Shallow Quantum Circuits <demos/qml/tutorial_learningshallow>
```

Comment:
I don't think it needs to be listed in the docs necessarily unless the demo adds a lot of Catalyst-specific material, but it can still live in the repo for us to test or as an example for others :)
```suggestion
```

Reply:
Perhaps it makes sense though to have it on the pennylane.ai/qml/demonstrations website?

Reply:
I'm not sure what the angle would be though, have all demos have a bare PennyLane and catalyst version? I wouldn't just want to throw out duplicates of existing demos without some cohesive plan 😅

Reply:
We _could_ update the original demo 🤔